File: Year 3/Android Java/Simple SMHI Warning App JavaVersion/GeoWarning/app/src/main/java/com/example/geowarning/model/GeoJSON.java
package com.example.geowarning.model;

import java.util.List;

public class GeoJSON {
    private String type;
    private List<Feature> features;
    private CRS crs;

    public CRS getCrs() {
        return crs;
    }

    public void setCrs(CRS crs) {
        this.crs = crs;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public List<Feature> getFeatures() {
        return features;
    }

    public void setFeatures(List<Feature> features) {
        this.features = features;
    }
}

File: Year 3/React Native/Simple SMHI Weather Warning application/my-second-project/models/GeoJSON.ts
import { CRS } from "./CRS";

class GeoJSON {

    type: string;
    features: any[];
    crs: CRS;

    constructor(type: string ='', features: any[] = [], crs?: CRS ){
        this.type = type;
        this.features = features;
        this.crs = new CRS();
    }

    getType(type: string | null){
        return this.type;
    }

    setType(type:string): void{
        this.type = type;
    }

    getFeatures(features: any[] | null){
        return this.features;
    }

    setFeatures(features: any[]): void{
        this.features = features;
    }

    getCrs(crs: CRS){
        return this.crs;
    }

    setCrs(crs: CRS): void{
        this.crs = crs;
    }
}